org.eclipse.debug.core/core/org/eclipse/debug/internal/core/LaunchConfigurationInfo.java

```diff
@@ -440,7 +440,7 @@ protected void setAttribute(String key, Object value) {
 	 *             if an exception occurs creating the XML
 	 */
 	@SuppressWarnings("unchecked")
-	protected String getAsXML() throws CoreException, IOException, ParserConfigurationException, TransformerException {
```

Comment:
you better keep this and call  getAsXML(System.lineSeparator()) then you do not need to touch all calls.

Reply:
Good idea. The code paths actually start through `LaunchManager.serializeDocument(..)` below, so I have overloaded that, which removed all changes to invocations except for the one that matters.